guides/common/modules/con_managing-content-views.adoc

```diff
@@ -34,10 +34,11 @@ The general workflow for creating Content Views for filtering and creating snaps
 
 . Create a Content View.
 . Add one or more repositories that you want to the Content View.
-. Optionally, create one or more filters to refine the content of the Content View.
-. Optionally, resolve any package dependencies for a Content View.
+. Optional:
+.. Create one or more filters to refine the content of the Content View.
+.. Resolve any package dependencies for a Content View.
```

Comment:
```suggestion
. Optional: Create one or more filters to refine the content of the Content View.
. Optional: Resolve any package dependencies for a Content View.
```

Reply:
So merging more optional steps into one is not desired in the documentation I take it?

Reply:
Yes. Also, _we/some_ prefer to not have subtasks within lists.